files/gitlab-cookbooks/gitlab/templates/default/database.yml.erb

```diff
@@ -12,3 +12,5 @@ production:
   host: <%= single_quote(@db_host) %>
   port: <%= @db_port %>
   socket: <%= single_quote(@db_socket) %>
+  socket: <%= single_quote(@db_sslmode) %>
+  socket: <%= single_quote(@db_sslrootcert) %>
```

Comment:
My bad, stupid mistake =(
The commit https://github.com/nawadanp/omnibus-gitlab/commit/77213ad3fe46983a79e725ecff84ca33809474b7 fix the typo